src/components/companies/detail/emissions-assessment/EmissionsAssessmentDialog.tsx

```diff
@@ -0,0 +1,99 @@
+import { Text } from "@/components/ui/text";
+import {
+  Dialog,
+  DialogContent,
+  DialogHeader,
+  DialogTitle,
+} from "@/components/ui/dialog";
+import type { paths } from "@/lib/api-types";
+
+type Assessment = paths["/emissions-assessment/"]["post"]["responses"][200]["content"]["application/json"]["assessment"];
+
+interface EmissionsAssessmentDialogProps {
+  isOpen: boolean;
+  onOpenChange: (open: boolean) => void;
+  assessment: Assessment | null;
+}
+
+export function EmissionsAssessmentDialog({
+  isOpen,
+  onOpenChange,
+  assessment,
+}: EmissionsAssessmentDialogProps) {
+  return (
+    <Dialog open={isOpen} onOpenChange={onOpenChange}>
+      <DialogContent className="max-w-3xl bg-black-2 text-white max-h-[90vh] overflow-hidden flex flex-col">
+        <DialogHeader>
+          <DialogTitle className="text-2xl font-light">
+            Emissions Assessment
```

Comment:
Longer term I think it makes sense to make these translatable as well. However I believe we don't have other internal admin pages localised either.